src/app/components/pages/TeachingOrders.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement, useEffect, useState } from "react";
+import { Col, Container, Row } from "reactstrap";
+import { Tabs } from "../elements/Tabs";
+import { PageFragment } from "../elements/PageFragment";
+import { TitleAndBreadcrumb } from "../elements/TitleAndBreadcrumb";
+import { MetaDescription } from "../elements/MetaDescription";
+
+enum TEACHING_ORDER_STAGE {
+    CORE_Y1 = "Core Y1",
+    CORE_Y2 = "Core Y2",
+    ADVANCED_Y1 = "Advanced Y1",
+    ADVANCED_Y2 = "Advanced Y2"
+}
+
+export const TeachingOrders = () => {
+    const TEACHING_ORDER_STAGES = Object.keys(TEACHING_ORDER_STAGE);
+    const [stageTab, setStageTab] = useState<typeof TEACHING_ORDER_STAGES[number]>(TEACHING_ORDER_STAGES[0]);
+    const [stageTabOverride, _setStageTabOverride] = useState<number | undefined>(TEACHING_ORDER_STAGES.indexOf(stageTab) + 1 || undefined);
+    const [fragmentId, setFragmentId] = useState<string>("");
+
+    const metaDescription = (stageTab === TEACHING_ORDER_STAGE.CORE_Y1 || stageTab === TEACHING_ORDER_STAGE.CORE_Y2)
+        ? "Discover our free Core computer science topics and questions. Learn or revise for your exams with us today."
+        : "Discover our free Advanced computer science topics and questions. Learn or revise for your exams with us today.";
+
+    const tabTitlesToContent: {[title: string]: ReactElement} = {};
+    for (const stageName of Object.values(TEACHING_ORDER_STAGE)) {
+        tabTitlesToContent[stageName] = <></>;
+    }
+
+    const stageTabs = <Tabs style={"buttons"} className={"mt-3"} tabContentClass={"mt-3"}
+        activeTabOverride={stageTabOverride}
+        onActiveTabChange={(n) => {
+            setStageTab(TEACHING_ORDER_STAGES[n-1]);
+        }}
+    >
+        {tabTitlesToContent}
+    </Tabs>;
+
+    useEffect(() => {
+        const sanitisedStageTab = stageTab.replace(" ", "_").toLowerCase();
```

Comment:
If you're using the above suggestion, we should use the same sanitisation function here too.
```suggestion
        const sanitisedStageTab = sanitiseStageTab(stageTab);
```